clover.rb

```diff
@@ -217,6 +217,12 @@ def last_mail_sent
     nil
   end
 
+  def vm_host_allowed?
+    vm_host_users = Config.allowed_vm_host_users
```

Comment:
For future, it may be a good idea to make this a database table. The reason is, when the team gets bigger, we'll have to make a new deployment so that new members can see the vm_host. For now, this may be better since the team is small and code footprint is very minimal.

Reply:
Makes sense. Maybe we can integrate with our authorization policies etc. It depends on our roadmap (hosted cheaper cloud, or enterprise first), and how VmHosts are managed.